src/content/docs/setup/systemd-hardening.mdoc

```diff
@@ -0,0 +1,109 @@
+---
+title: systemd hardening
+slug: systemd-hardening
+description: >-
+  Run Vikunja as a non-root user with systemd for additional security,
+  and optionally apply sandboxing to further limit access.
+---
+
+When [installing](/docs/installing) Vikunja on a systemd-based system, the service may be configured to run as `root`. It is generally a good idea to run services with the least amount of privileges they need. This guide shows how to run Vikunja as its own dedicated user and how to optionally add extra security restrictions using systemd.
+
+## Create a dedicated user
+
+Create a system account for Vikunja that will not be able to log in interactively and will only be used to run the service:
+
+```bash
+sudo useradd --system --home /var/lib/vikunja --shell /usr/sbin/nologin vikunja
+```
+
+Next, ensure Vikunja has its own directories for data, logs, and configuration, and that the new user owns them:
```

Comment:
I would assume the directories for logs are only needed when logging to files? If yes, please add clarify this.